java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/searcher/TestFrameworkSearcher.java

```diff
@@ -33,7 +40,12 @@
 
     SearchPattern getSearchPattern();
 
+    boolean annotates(IAnnotationBinding[] annotations, String annotationName);
```

Comment:
Batch api is more flexible.

boolean findAnnotation(IAnnotationBinding[] annotations, String[] names)